docs/components/pagination/index.html

```diff
@@ -0,0 +1,146 @@
+        <button class="hxBtn">5</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    </div>
+    {% code 'html' %}
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
+        <button class="hxBtn">2</button>
+        <button class="hxBtn">3</button>
+        <button class="hxBtn">4</button>
+        <button class="hxBtn">5</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    {% endcode %}
+  </section>
+
+  <section>
+    <h2 class="hxSectionTitle" id="middle-page">Middle Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn">
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn">8</button>
+        <button class="hxBtn">9</button>
+        <button class="hxBtn" aria-current="true">10</button>
+        <button class="hxBtn">11</button>
+        <button class="hxBtn">12</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    </div>
+    {% code 'html' %}
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn">
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn">8</button>
+        <button class="hxBtn">9</button>
+        <button class="hxBtn" aria-current="true">10</button>
+        <button class="hxBtn">11</button>
+        <button class="hxBtn">12</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    {% endcode %}
+  </section>
+
+  <section>
+    <h2 class="hxSectionTitle" id="last-page">Last Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn">
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn">1</button>
```

Comment:
Update page numbers to span from 16-20. This makes the progression in the docs go from "the first 5 pages" (1-5) to the "middle 5 pages" (8-12) then the "last 5 pages" (16-20).